apps/system/js/ftu_ping.js

```diff
@@ -25,7 +25,8 @@
   const DEFAULT_PING_TIMEOUT = 60 * 1000;
 
   // Used by the telemetry server to help identify the payload format
-  const PING_PAYLOAD_VERSION = 3;
+  const TELEMETRY_VERSION = 1;
```

Comment:
It looks like the common code is now inserting the "info" section as described in comment 1 of bug 1037495 (for both the app usage ping and the ftu ping).  Is that right?  If that's the case, then setting this version back to 1 is correct.  If the ftu pings will continue to look like they did in bug 987232, the version should stay at 3.